src/components/GeneFrequencyTable.tsx

```diff
@@ -18,7 +17,7 @@ import GeneTumorTypeFrequencyDecomposition from "./GeneTumorTypeFrequencyDecompo
 import { comparePenetrance } from './Penetrance';
 
 import "react-table/react-table.css";
-import "./FrequencyTable.css";
+import "react-variant-view/dist/styles.css";
```

Comment:
Do we need this import? Seems like we already import it in App.tsx:

https://github.com/knowledgesystems/signal/blob/354c77e5870498ec1edc7b2170c000b8f21b80ac/src/App.tsx#L8

Reply:
@onursumer oops, yes we already imported it in App.tsx, removed